client/src/theme/Components/_toggle_primary.scss

```diff
@@ -0,0 +1,40 @@
+@mixin togglePrimary($height) {
```

Comment:
Since this style/component has its own partial and isn't being used in multiple contexts, we don't need a mixin here.